src/sidebar/components/Annotation/AnnotationShareControl.tsx

```diff
@@ -116,8 +111,8 @@ function AnnotationShareControl({
   return (
     // Make the container div focusable by setting a non-null `tabIndex`.
     // This prevents clicks on non-focusable contents from "leaking out" and
-    // focusing a focusable ancester. If something outside of the panel gains
-    // focus, `usePopoverShouldClose`'s focus listener will close the panel.
+    // focusing a focusable ancestor. If something outside the panel gains
+    // focus, Popover's internal logic will close the panel.
     // "Catch focus" here to prevent this.
     // See https://github.com/hypothesis/client/issues/5196
     <div className="relative" ref={shareRef} tabIndex={-1}>
```

Comment:
I tried removing the `tabIndex` prop here and the copy-to-clipboard buttons still works in all browsers, including Safari 16 that was mentioned in the linked issue.

The reason for this I believe is related to Lyza's helpful comment in the issue:

> The copy-to-clipboard code, triggered by button clicks in non-Safari browsers, steals focus. It sets focus on a temporary textarea element and does not restore it. This also causes the panel to close (as it causes a focus event outside of the panel).

The copy-to-clipboard logic no longer uses a hidden textarea, so this issue doesn't apply any more. Hence we can remove this workaround.